micrometer-api/src/main/java/io/micrometer/api/instrument/observation/NoopObservation.java

```diff
@@ -13,11 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.micrometer.api.instrument;
+package io.micrometer.api.instrument.observation;
 
-import io.micrometer.api.instrument.observation.Observation;
-import io.micrometer.api.lang.Nullable;
+import io.micrometer.api.instrument.Tag;
 
+/**
+ * No-op implementation of {@link Observation} so that we can disable the instrumentation logic.
+ *
+ * @author Jonatan Ivanov
+ * @author Tommy Ludwig
+ * @author Marcin Grzejszczak
+ *
+ * @since 2.0.0
+ */
 public class NoopObservation implements Observation {
```

Comment:
Fine to consider this separate from this PR, but:
Is there any point in this being public API? We aren't expecting people to extend this, are we?

Reply:
I need to take a look but I think it is public because the `MeterRegistry` is using it (we can make it final `though`).